src/index.ts

```diff
@@ -158,6 +159,23 @@ class Storyblok {
 		})
 	}
 
+	public customFetch(
+		slug: string,
+		params?: ISbStoriesParams,
+		fetchOptions?: object
+	): Promise<any> {
+		if (fetchOptions && Object.keys(fetchOptions).length > 0) {
+			this.fetchOptions = fetchOptions
```

Comment:
@thiagosaife why storing the `fetchOptions` in the instance of the client? I see a potential issue here.

Imagine this scenario:
- User calls `customFetch` passing some options => those options willl apply
- Then user performs a call with no options => as they're stored in the client instance, they will apply?